packages/components/src/ui/utils/test/create-component.js

```diff
@@ -1,70 +0,0 @@
-/**
```

Comment:
Not sure if these tests have any value even without the `createComponent` function — should we potentially keep / move them, instead of deleting them?

Reply:
Hmmm, I guess if we wanted to keep these tests we could create a function that generates a generic test suite for any component that follows that general pattern of passing hook props to `View`. Not sure if there's much value in that though.

Reply:
Yeah, not sure if there's much value in doing that. I guess we can just delete these tests.